shap/explainers/tree.py

```diff
@@ -1192,7 +1192,7 @@ def __init__(self, cb_model):
         self.loaded_cb_model = json.load(open("cb_model.json", "r"))
 
         # load the CatBoost oblivious trees specific parameters
-        self.num_trees = self.loaded_cb_model['model_info']['params']['boosting_options']['iterations']
+        self.num_trees = self.num_trees = len(self.loaded_cb_model['oblivious_trees'])
```

Comment:
Might want to remove the double `self.num_trees = self.num_trees =`

Reply:
Good point. Done.